app/src/services/attestation.tsx

```diff
@@ -39,6 +64,70 @@ type AttestationProviderParams = {
 
 export type EventListenerFn = (event: BaseEvent) => void
 
+const attestationCredDefIds = [
+  'J6LCm5Edi9Mi3ASZCqNC1A:3:CL:109799:dev-attestation',
+  'NxWbeuw8Y2ZBiTrGpcK7Tn:3:CL:48312:default',
+]
+
+// change this URL to a multi use connection from your traction instance for testing
+const attestationInviteUrl =
```

Comment:
Should this find its way into .env or contstants?

Reply:
Eventually but not yet, cause you change it fairly frequently for development so it's nice to not have to rebuild the entire project when you do